main.py

```diff
@@ -1,81 +1,97 @@
 import pymongo
 from pprint import pprint
-import Book
-import Review
+import Book  # Importing Book module, assuming it contains the Book class
+import Review  # Importing Review module, assuming it contains the Review class
 from pymongo import MongoClient
 
 try:
+    # Attempting to establish a connection to the MongoDB server
     client = MongoClient('localhost', 27017)
     print("\nSuccessfully connected!")
 except:
+    # If connection fails, print an error message
     print("\nConnection failed.")
-    
+
+# Accessing the 'main' database
 db = client.main
-books = db.book
-reviews = db.review
+books = db.book  # Assuming there's a collection named 'book' in the 'main' database
+reviews = db.review  # Assuming there's a collection named 'review' in the 'main' database
 
+# Dropping collections to start fresh
 books.drop()
 reviews.drop()
```

Comment:
Kiekvieną kartą pasileidus programą knygų ir atsiliepimų duomenys bus išvalyti iš duomenų bazės, ar mes to ir siekiame? Ar tai yra tiesiog pasilikusios debug'inimo eilutės? Jeigu to siekiame pravartu būtų palikti komentarą kodėl naudojamas toks metodas?